packages/patterns-react/src/patterns/index.js

```diff
@@ -8,4 +8,5 @@
 export * from './LeadSpace';
 export * from './ListSection';
 export * from './SimpleLongForm';
+export * from './CardsWithoutImages';
```

Comment:
@sannminwin minor thing, can this be alphabetical in the export list?

Reply:
fixed